### ISSUE TITLE
Focus Console Output command not working when virtual scrolling feature enabled

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7788]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.771
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Generate some output in the console, e.g. `cat("Hello")`
- Use the "Focus Console Output" command from command-palette or via <kbd>Alt+Shift+2</kbd>

### Describe the problem in detail
- Keyboard focus does not go to the console output, it remains on the console input control

### Describe the behavior you expected
- Keyboard focus goes to the console output region

To test this, you'll need to have a screen reader running, or at least be monitoring the focused element via a Chrome devtools live expression for `document.activeElement`.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
This problem was introduced by the virtual console feature (#7568).

First thought is to disable the virtual console feature when screen reader mode is on, but problem remains even with this feature off. Specifically:

In 1.3, console output HTML looks like this:
```
<pre tabindex="0" aria-label="Console Output" role="document"> <!-- this wraps the entire console output region -->
    <span tabindex=-1 role="document"> <!-- a "chunk" of output, e.g. from executing a command in the console -->
        <span>actual output seen in console</span>
        <span>more output from this command</span>
        <!-- and so on -->
    </span>
    <span tabindex=-1 role="document"> <!-- another "chunk" of output, e.g. from another command -->
        <span>actual output seen in console from another command</span>
        <span>more output from this second command</span>
        <!-- and so on -->
    </span>
</pre>
```
`tabIndex=-1` is needed on each chunk’s wrapper so we can set focus to it via code. `role="document"` is needed both on the pre and the chunk spans for JAWS to correctly announce it and enter virtual cursor mode.

In 1.4, with virtual scroller off, the output looks something like:
```
<pre tabindex="0" aria-label="Console Output" role="document"> <!-- SAME -->
    <span tabindex=-1 role="document"></span> <!-- DOESN'T CONTAIN THE CHUNK'S OUTPUT -->
    </span>
    <span>actual output seen in console</span> <!-- NOT NESTED IN PREVIOUS SPAN -->
    <span>more output from this command</span>
    <span>actual output seen in console from another command</span>
    <span>more output from this second command</span>
    <!-- and so on -->
    </span>
</pre>
```
In 1.4, there are sometimes additional `<spans>` in there, some just plain, some with the `tabindex=-1`.

**Comment by kfeinauer** [comment]
Verified fixed on RSP 1.4.940-1
